src/model/modeling/adapter.py

```diff
@@ -30,22 +30,37 @@
 ice = ICE()
 
 
-def adapt_from_medium(model, medium):
+def apply_medium(model, medium):
     """
-    Returns a list of operations to apply the given medium to the given model
-
-    :param model: cobra.Model
-    :param medium: list of dictionaries of format
-        {'id': <compound id (<database>:<id>, f.e. chebi:12345)>, 'concentration': <compound concentration (float)>}
+    Apply a medium to a metabolic model.
+
+    The medium is applied by adding salt mappings, resolving the corresponding
+    exchange reaction for the resulting medium compounds, setting their uptake
+    rate, and then applying that to the model.
+
+    Parameters
+    ----------
+    model: cobra.Model
+    medium: list(dict)
+        The medium definition, a list of dicts matching the `MediumCompound`
+        schema.
+
+    Returns
+    -------
+    tuple (operations, errors)
+        Operations is a list of model operations necessary to apply the medium
+        to the model. See also the `Operations` schema.
+        If errors is not an empty list, it was not possible to apply the medium.
+        Errors then contains a list of string messages describing the
+        problem(s).
     """
-
     operations = []
     errors = []
 
     # Detect salt compounds
     chebi_ids = [c['id'] for c in medium]
     for compound in chebi_ids:
-        chebi_id = compound.replace('chebi:', '')
+        chebi_id = compound.replace('CHEBI:', '')
```

Comment:
I feel like the `salts.csv` content should be prepended with CHEBI: since that's an integral part of the identifier but I can't be bothered myself right now.